lib/wallet/wallet_dashboard.dart

```diff
@@ -92,6 +99,49 @@ class _WalletDashboardState extends State<WalletDashboard> {
     );
 
     widgets.add(paymentHistoryList);
+    if (Platform.isAndroid || Platform.isIOS || true) {
```

Comment:
```suggestion
```
no need for this `if` anymore, if it always evaluates to `true` 😉

Reply:
ups.. I added true only for testing 😅 good catch 👍